### PR TITLE
Adding Health controller

### PR DESCRIPTION
**Pull request by therobbiedavis**
# Added
- Added: Added API endpoint for a health check to streamline docker healthy status.

### CODE REVIEW

File: API/Controllers/HealthController.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+namespace API.Controllers;
+
+[AllowAnonymous]
+public class HealthController : BaseApiController
+{
+
+    [HttpGet("")]
+    public string GetHealth()
+    {
+        return "ok";
```
Comment (majora2007) [inline API/Controllers/HealthController.cs:15]: this needs to be return Ok("Ok"); and the result type is ActionResult

**Review by majora2007 — CHANGES_REQUESTED** [review]
Please address comments

File: API/Controllers/HealthController.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers;
+
+[AllowAnonymous]
+public class HealthController : BaseApiController
+{
+
+    [HttpGet("")]
```
Comment (majora2007) [inline API/Controllers/HealthController.cs:12]: Oh and remove the (""), as it's implied

Comment (therobbiedavis) [reply API/Controllers/HealthController.cs]: Fixed in [e22e75f](https://github.com/Kareadita/Kavita/pull/1480/commits/e22e75ff9d7e5ea4f6539018fa53f45ae341ce1e)

File: API/Controllers/HealthController.cs (hunk 1 above)
Comment (therobbiedavis) [reply API/Controllers/HealthController.cs]: Fixed in [e22e75f](https://github.com/Kareadita/Kavita/pull/1480/commits/e22e75ff9d7e5ea4f6539018fa53f45ae341ce1e)

**Review by majora2007 — APPROVED** [review]

**Pull request merged**